index.md

```diff
@@ -1,47 +1,42 @@
-|[`harbor-scanner-sysdig-secure`](/charts/harbor-scanner-sysdig-secure) | Deploys the Harbor Scanner Adapter to enable Harbor to use Sysdig Secure scanning engine to analyze the container images managed by the platform. |
-|[`registry-scanner`](/charts/registry-scanner) | Deploys the Sysdig Registry Scanner as a scheduled cronjob in your Kubernetes cluster. |
-|[`sysdig-mcm-navmenu`](/charts/sysdig-mcm-navmenu) | Deploys services and ingress required to create Multi-Cloud Management menu in Sysdig. |
-|[`sysdig-stackdriver-bridge`](/charts/sysdig-stackdriver-bridge) | Deploys the the Sysdig Stackdriver Bridge application to enable reading and sending audit logs from Stackdriver to Sysdig agent. |
+Additional Sysdig Secure components provide scanning for vulnerabilities in various parts of the environment and the CI/CD lifecycle.
 
+| Parent Chart                                   | Subcharts                                              | Description                                                  |
+| ---------------------------------------------- | :----------------------------------------------------- | ------------------------------------------------------------ |
+| [`sysdig-deploy`](/charts/sysdig-deploy)       |                                                        | Installs, configures, and upgrades the Sysdig Agent and a variety of Sysdig Secure components. The parent chart deploys multiple child charts. |
+|                                                | [`sysdig-agent`](/charts/agent)                        | Processes syscall events, metrics and creates capture files. It provides detailed visibility into container and host activity, helping to detect and prevent threats. |
+|                                                | [`node-analyzer`](/charts/node-analyzer)               | The `node-analyzer` deploys the following as one daemonset in a cluster: <br>`host-scanner`<br>`runtime-scanner`<br>`kspm-analyzer` |
+|                                                | [`kspm-collector`](/charts/kspm-collector)             | Collects Kubernetes resource manifests and sends them to be evaluated against Sysdig compliance policies. The scan results are displayed in Sysdig Secure [Compliance](https://docs.sysdig.com/en/compliance/) and [Inventory ](https://docs.sysdig.com/en/docs/sysdig-secure/inventory/)UI. |
+|                                                | [`host-scanner`](/charts/node-analyzer)                | Scan [hosts](https://docs.sysdig.com/en/docs/sysdig-secure/vulnerabilities/runtime/host-scanning/) to identify and prioritize vulnerabilities with rich context. |
+|                                                | [`runtime-scanner`](/charts/node-analyzer)             | Scan Kubernetes [workloads](https://docs.sysdig.com/en/docs/sysdig-secure/vulnerabilities/runtime/) in runtime to identify and prioritize vulnerabilities with rich context. |
```

Comment:
Having these called out in the `Subcharts` column is a bit misleading as they are not their own charts, but rather a part of the `node-analyzer` chart. It's possible someone would get confused clicking on the `host-scanner` link and immediately landing on a page called `Node Analyzer`. 🤷🏻

Reply:
Good point. I wonder if it's still worthwhile to call out the components but begin the description with "Component deployed in node-analyzer chart to...[scan hosts etc]" and ""Component deployed in node-analyzer chart to...[scan Kubernetes workloads etc]"